repo/packages/R/riak-kv/0/package.json

```diff
@@ -1,15 +1,15 @@
 {
```

Comment:
This was not done with a `git mv` - but because the contents are similar enough to a deleted file `git` assumes it is such.

Perhaps if I move the deletion into a separate commit it will not do this. I will try it out.